### PR TITLE
style my work

### PR DESCRIPTION
**Pull request by osbentech**
This is a React/redux project named bookStore App.
It will enable user to add, delete and edit books upon completion, for now it is a work in progress
below are few functionalities I have implemented:

    connect my App to Api
    Add an Action Creator, that returns a function.
    Fetch list of books
   implement styles

### CODE REVIEW

File: src/components/Book.css [hunk 1]
```diff
@@ -15,11 +15,10 @@
   margin: auto 0 auto 40px;
 }
```
Comment (OLIPLICHE) [inline src/components/Book.css:17]: ## Your final design should match Zeplin's design:
- Features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented
- Kindly [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project)
- [Zeplin's design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png)
- 

Capture


- Your design:
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @osbentech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The App is working fine. 
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.css [hunk 2]
```diff
@@ -12,14 +12,12 @@
 
 .book-holder {
   text-align: left;
-  margin: auto 0 auto 40px;
 }
```
Comment (Mihndim2020) [inline src/components/Book.css:20]: - [ ] As requested by the previous code reviewer, which is a requirement for the project, please make sure your implementation matches the [design ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png)

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @osbentech , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @osbentech,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

- [ ] Just as the previous reviewers requested, your design is yet to match the original design given in the project requirements. I made a screenshot marking the points you should consider adjusting your styles.
- [ ] The Title/Logo of the app should be spaced backward just I the arrow in the screenshot
- [ ] The title input fields as well should be spaced to the left a little
- [ ] The Add Book button should be aligned with the cards
- [ ] Please consider adding the user icon just as it's in the original design

**Your Design**

image

**The original Design**

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @osbentech,

✔️ Nice styling

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**